src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json

```diff
@@ -0,0 +1 @@
+{"case_id":1,"case_number":"1","courthouse":"some-courthouse","judges":["1judge1"],"hearing_date":"2023-01-01","urgency":"STANDARD","request_type":"SENTENCING_REMARKS","transcription_start_ts":"2023-01-01T12:00:00Z","transcription_end_ts":"2023-01-01T12:00:00Z"}
```

Comment:
I'm not sure why urgency and request_type are in upper snake case - it might just be down to some funny mismatch we have in our standing data for h2 perhaps? I just thought it would be like `Sentencing remarks`